src/js/components/icons/status/Disabled.js

```diff
@@ -1,22 +1,26 @@
 // (C) Copyright 2014-2015 Hewlett-Packard Development Company, L.P.
 
 var React = require('react');
+var IntlMixin = require('../../../mixins/GrommetIntlMixin');
 
 var Disabled = React.createClass({
 
+  mixins: [IntlMixin],
+
   render: function() {
     var className = 'status-icon status-icon-disabled';
     if (this.props.className) {
       className += ' ' + this.props.className;
     }
     return (
-      <svg className={className} viewBox="0 0 24 24" version="1.1">
-        <g className={"status-icon__base"}>
-          <rect x="0" y="0" width="24" height="24" stroke="none"></rect>
+      <svg className={className} viewBox="0 0 24 24" role="img" aria-labelledby="title-icon-disabled" version="1.1">
+        <title id="title-icon-disabled">{this.getGrommetIntlMessage('Disabled')}</title>
```

Comment:
Are the tools smart enough to handle multiple elements in the DOM with the same id? I ask because we can easily have multiple status icons on a screen and usually the id attribute is expected to be unique within the DOM.